projects/fsastorefrontlib/src/cms-components/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,212 @@
+              <li class="item text-break text-center text-uppercase">
+                <div class="number notice semi-bold">
+                  {{
+                    customerAssets.claims?.claims
+                      ? customerAssets.claims?.claims?.length
+                      : '0'
+                  }}
+                </div>
+                <span class="description">{{
+                  'claim.claims' | cxTranslate
+                }}</span>
+              </li>
+              <li class="item text-break text-center text-uppercase">
+                <div class="number notice semi-bold">
+                  {{
+                    customerAssets.policies?.insurancePolicies
+                      ? customerAssets.policies?.insurancePolicies?.length
+                      : '0'
+                  }}
+                </div>
+                <span class="description">{{
+                  'policy.policies' | cxTranslate
+                }}</span>
+              </li>
+              <li class="item text-break text-center text-uppercase">
+                <div class="number notice semi-bold">
+                  {{
+                    customerAssets.quotes?.insuranceQuotes
+                      ? customerAssets.quotes?.insuranceQuotes?.length
+                      : '0'
+                  }}
+                </div>
+                <span class="description">
+                  {{ 'fscommon.quotesAndApplications' | cxTranslate }}
+                </span>
+              </li>
+            </ul>
+            <div class="dashboard-overview-assets mt-3">
+              <div
+                *ngFor="let asset of customerAssets | keyvalue; let i = index"
+                [ngClass]="assetSelected === asset.key ? 'active' : ''"
+                class="asset bg-white p-2 align-items-center"
+                role="button"
+                [tabindex]="i + 1"
+                (click)="
+                  showAssetList(
+                    asset.key === 'quotes'
+                      ? customerAssets.quotes?.insuranceQuotes
+                      : asset.key === 'policies'
+                      ? customerAssets.policies?.insurancePolicies
+                      : asset.key === 'claims'
+                      ? customerAssets.claims?.claims
+                      : '',
+                    asset.key
+                  )
+                "
+                (keydown.enter)="
+                  showAssetList(
+                    asset.key === 'quotes'
+                      ? customerAssets.quotes?.insuranceQuotes
+                      : asset.key === 'policies'
+                      ? customerAssets.policies?.insurancePolicies
+                      : asset.key === 'claims'
+                      ? customerAssets.claims?.claims
+                      : '',
+                    asset.key
+                  )
+                "
+              >
+                <span
+                  [ngClass]="
+                    asset.key === 'quotes'
```

Comment:
Maybe here you could extend icon configuration enum `FS_ICON_TYPE` with QUOTES, POLICIES AND CLAIMS, add one variable `icontType = FS_ICON_TYPE` and use it through template like this:

`[ngClass]="iconType[asset.key.toUpperCase()]"`

Something like that...

Reply:
Yup, good idea, just we cannot use it with `ngClass` rather with `cx-icon`. But since we plan to implement some sort of config approach, this should also be a part of it. We should disscuss how best to utilize `cx-icon` for this.

Reply:
Are you sure we cannot use it with `ngClass`? I don't see an obstacle for that, because `iconType[asset.key.toUpperCase()]` returns `string`.

Reply:
Of course we could use it technically as you described, but we should follow the Spartacus approach and use `iconType`s with `cx-icon`s

Reply:
Ok, if you think that Spartacus gives best practice just follow them.